hs_core/tasks.py

```diff
@@ -475,11 +471,9 @@ def create_bag_by_irods(resource_id):
             for fname in bagit_files:
                 if istorage.exists(fname):
                     istorage.delete(fname)
-            logger.error(ex.stderr)
-            return False
+            raise SessionException(-1, '', ex.stderr)
```

Comment:
Not sure what the -1 is for.  Couldn't you just call raise to re-raise the SessionException?